environments/staging/app-processes.yml

```diff
@@ -34,10 +34,8 @@ celery_processes:
       concurrency: 1
     ucr_queue:
       concurrency: 1
-    user_import_queue:
+    user_import_queue, ush_background_tasks:
```

Comment:
could you keep these queues defined on separate lines? Minimizes diffs which is a good thing in this context